frontend/src/components/SearchComponent.jsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button, Input } from "@mui/material";
+import SearchIcon from '@mui/icons-material/Search';
+
+const SearchComponent=()=>{
+  const handleSearch = async () => {
```

Comment:
There's no real need to declare the _handleSearch_ functions as an async function. Consider using 2 _.then_ lambdas instead